src/sections/Community/Members-grid/index.js

```diff
@@ -63,17 +63,14 @@ const MembersGrid = props => {
 
               <p className="text-gray">Would you like a member profile?</p>
               <div className="">
-                <VintageBox right={true} vintageOne={true}>
+                <Button primary className="banner-btn memberProfileBtn" title="Let Layer5 Community Managers Know" url="mailto:community@layer5.io?subject=Layer5 Community Membership" external="true">
```

Comment:
Also, while you are here update the `external` prop to `external={true}`

Reply:
Updated!